.github/workflows/argus-docker-build.yaml

```diff
@@ -65,15 +67,13 @@ on:
           If ANY of the labels are found on the PR, the manifest will be updated.
         required: false
         type: string
-        default: 'stack'
+        default: 'stack,stack-*'
```

Comment:
did someone request stack-*?

Reply:
not `stack-*` explicitly but we've seen the metahub project want to use `stack-science` and edu point solutions needing `stack-notifications`

I'm not attached to this change but thought it might be nice to not have to specify this input in most cases